python/flatbuffers/builder.py

```diff
@@ -21,8 +21,9 @@
 from . import compat
 from .compat import range_func
 from .compat import memoryview_type
+from .compat import import_numpy, NumpyRequiredForThisFeature
```

Comment:
does this still allow this file to be used with Python installations that don't have numpy?

Reply:
Yes,

the compat version of `import_numpy` checks first if numpy is an installed module and then assigns `np` to be either the numpy module or `None`